libs/core/schematics/migrations/migration-0-37/index.ts

```diff
@@ -32,7 +32,7 @@ function removeStylesFromConfig(): Rule {
                     ];
 
                     const stylesToRemove = [
-                        'node_modules/fundamental-styles/dist/icon.css',
+                        '~fundamental-styles/dist/icon.css',
```

Comment:
What happened here?

Reply:
was playing with paths, accidentally affected this file